<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <family>Script MT Bold</family>
        <pointsize>14</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Segoe Print&quot;;
font: 700 14pt &quot;Script MT Bold&quot;;
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="login_PAGE0">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(71, 1, 6);</string>
       </property>
       <widget class="QPushButton" name="SING_UP_firstpage">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>300</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 italic 14pt &quot;Segoe UI&quot;;
background-color: rgb(255, 217, 140);
</string>
        </property>
        <property name="text">
         <string>login</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>661</width>
          <height>291</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/new/prefix1/image/asterisk-kwon-q_gjDWf9ths-unsplash.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="login_front_page1">
       <property name="styleSheet">
        <string notr="true">gridline-color: rgb(97, 97, 0);
background-color: rgb(250, 255, 194);
border-color: rgb(255, 245, 165);</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Script MT Bold</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="login_email_line">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(207, 255, 241);</string>
        </property>
        <property name="placeholderText">
         <string>               @gmail.com</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="login_password_line">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>150</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>.......................password...........................</string>
        </property>
       </widget>
       <widget class="QPushButton" name="login_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>210</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>login</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>40</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 239, 175);
border-color: rgb(0, 0, 0);
font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Sign up</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Signup_email_line">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>130</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>                   @gmail.com</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="signup_pass_line">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>170</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>120</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Script MT Bold</family>
          <pointsize>14</pointsize>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;
</string>
        </property>
        <property name="text">
         <string>login</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sign_up_p2">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>310</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>sign up</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="signup_address_line">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>210</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>flat no, area , district</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>210</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="signup_phone_line">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>260</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>+880</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>260</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="signup_name_line">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>90</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>95</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QPushButton" name="close_app">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>330</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>close app</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page2">
       <widget class="QPushButton" name="order_book">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Order book</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sell_book">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Sell book</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_existing_book">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string> book quantity</string>
        </property>
       </widget>
       <widget class="QPushButton" name="back">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>210</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>logout</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>311</width>
          <height>261</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/new/prefix1/image/janko-ferlic-sfL_QOnmy00-unsplash.jpg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="seller_add_page4">
       <widget class="QLineEdit" name="book_name">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="book_writer">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="add_book_quantity">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="book_code">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>190</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Book Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Book writer</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>155</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Add Quantity</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>book code</string>
        </property>
       </widget>
       <widget class="QPushButton" name="post_sell_book">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>340</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Post</string>
        </property>
       </widget>
       <widget class="QPushButton" name="back_page_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>back</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>book price</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="book_price">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>243</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="seller_page4_listview">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>40</y>
          <width>381</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="book_list_page4">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>210</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>list</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="search_page3">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>260</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>Book Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>300</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>book code</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="book_sell_name">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>260</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="book_sell_code">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>300</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="back_page_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>51</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buy_page_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>340</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>buy</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>541</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="booklist">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>230</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Script MT Bold&quot;;</string>
        </property>
        <property name="text">
         <string>book list</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="addexixtingbok">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/new/prefix1/image/asterisk-kwon-q_gjDWf9ths-unsplash.jpg);</string>
       </property>
       <widget class="QLineEdit" name="addexistingbook_code">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>250</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>250</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>book code</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>290</y>
          <width>61</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>quantity</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>290</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addexistingbook">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>330</y>
          <width>80</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="back_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>61</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>back</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>561</width>
          <height>192</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 255, 194);</string>
        </property>
       </widget>
       <widget class="QPushButton" name="list_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>20</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>book list</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
